VndbSharp/VndbUtils.cs

```diff
@@ -0,0 +1,87 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.IO.Compression;
+using System.Net;
+using System.Text;
+using System.Threading.Tasks;
+using Newtonsoft.Json;
+using VndbSharp.Structs.Models.Dumps;
+
+namespace VndbSharp
+{
+    public class VndbUtils
+    {
+        protected DateTime LastTime = DateTime.Now;
+        protected static Boolean FirstRun = true;
```

Comment:
the DateTime by default use the minimum value instead of null. I can make it the DateTime object nullable if you wish, and remove the FirstRun Boolean.
Just explaining why I did it in two variables.